src/metadataGeneration/tsoa.ts

```diff
@@ -43,8 +43,7 @@ export namespace Tsoa {
   }
 
   export interface Security {
-    name: string;
-    scopes?: string[];
+    [key: string]: string[];
```

Comment:
Too bad we're losing the _Security_ object true definition. Are we ?

Reply:
Sorry, I don't know what that means.

Reply:
I think the idea of the comment is that a security was "a name and a scope list" now it's "a hashmap of string array." Therefore, on the autocomplete and code comprehension, we've lost something.

Reply:
I guess the alternative would be to have a return type in the `methodGenerator:233` of `[[Tsoa.Security]]`, where the inner array denotes a combination of AND and the outer array denotes a combination of OR. I thought the way I did it would be easier to grasp, but yes, some information is lost. Do you guys think I should do it the other way? At least `Validators` uses a similar pattern, but I admit, that it's clearer there what the parts mean.

Reply:
So that's a matter of choosing between code readability and ease of use ? I'm betting on "ease of use". I know it might not go in the way of maintainability, but if it's easier to grasp for an end user ...

Reply:
How does this make a difference for an end user? They're just using the decorators. They won't even see this internal representation.